api.ts

```diff
@@ -187,10 +188,17 @@ export interface SourceFileConfiguration {
     readonly compilerPath?: string;
 
     /**
-     * Arguments for the compiler.
+     * Arguments for the compiler. These arguments should not require shell parsing, so should not include any

```

Comment:
consider making the statement stronger.
e.g.
`These arguments will not be processed by the shell and should not include any shell variables, quoting or escaping.`